faasmtools/build.py

```diff
@@ -137,12 +137,13 @@
     "-Xlinker --stack-first",
     "-Xlinker --export=__heap_base",
     "-Xlinker --export=__data_end",
+    "-Xlinker --export=free",
+    "-Xlinker --export=malloc",
     "-Xlinker --export={}".format(FAASM_WASM_CTORS_FUNC_NAME),
     "-Xlinker --export=__stack_pointer",
     "-Xlinker --max-memory={}".format(FAASM_WASM_MAX_MEMORY),
     "-Xlinker --features=mutable-globals,simd128",
     "-Wl,-z,stack-size={} -Wl".format(FAASM_WASM_STACK_SIZE),
-    "-Wl,--initial-memory={}".format(FAASM_WASM_INITIAL_MEMORY_SIZE),
```

Comment:
The WASM linker already sets the initial memory to the minimum value, so I don't think we need to turn this knob.

Reply:
This reminds me of some other changes we made when building WAMR, can you have a look at the args we pass in the WAMR `CMakeLists.txt` and make sure they're still relevant? https://github.com/faasm/faasm/blob/main/src/wamr/CMakeLists.txt#L35